tests/performance-tests/src/tests/testData.ts

```diff
@@ -0,0 +1,10 @@
+const testData = {
+  themeName: 'UI test theme - Performance tests',
+  topicName: 'UI test topic - Performance tests',
+  maxImportWaitTimeMillis: 2 * 60 * 60 * 1000,
+};
+export default testData;
+  themeName: 'UI test theme - Performance tests',
```

Comment:
Think the suggestion has gone a bit awry here

Reply:
Cheers, manually fixed!  Shame, it's a good idea for a mechanism - I may have used it incorrectly...